index.html

```diff
@@ -0,0 +1,49 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <title>Hamma's Portflolio</title>
+</head>
+
+<body>
+    <div class="wrapper">
+        <nav class="nav-bar">
+            <p>Flex Box</p>
+            <img src="img/Enabled.png" alt="Logo">
+        </nav>
+        <header class="header">
+            <span class="header-text">I’m Hamma
+                Glad to see you!</span>
+            <p>
```

Comment:
- [ ] Kindly use `<h1>` for the main title to improve SEO [read more](https://www.reliablesoft.net/h1-tag/#h1-tag)